pynamodb/tests/test_signals.py

```diff
@@ -0,0 +1,106 @@
+import sys
+
+import mock
+import pytest
+
+from pynamodb.connection import Connection
+from pynamodb.connection.signals import _FakeNamespace
+from pynamodb.connection.signals import pre_boto_send, post_boto_send
+
+try:
+    import blinker
+except ImportError:
+    blinker = None
+
+
+pytestmark = pytest.mark.skipif(
```

Comment:
Why would we support running tests without `blinker` available? Since we control the requirements, this doesn't need to be a concern. We should instead additionally test the behavior where the system _thinks_ blinker is not installed to ensure that works as expected